heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,25 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean consolePrompt(int oldContainerCount, int newContainerCount) {
+    String fmt =
+        "The present aurora job has %d containers. After update there will be %d containers. ";
+    Console c = System.console();
```

Comment:
In general, it is a bad idea to require a console to be assigned as this very much impacts the flexibility of your application. Many ways of invoking Java will not assign a console (i.e. `System.console()` returns null), and your application is unusable in those instances.
(https://stackoverflow.com/questions/20982664/why-does-system-console-return-null-for-a-command-line-app)

You can always use `System.in` and `System.out` instead, as follows:

```
String qq;
Scanner scanner = new Scanner(System.in);
qq = scanner.nextLine();
System.out.println(qq);
```

Reply:
updated